bootstrap/Dockerfile

```diff
@@ -1,16 +1,5 @@
-# bootstrapper-builder contains the tool chain we need to build the image.
-# This primarily means glide.
-ARG GOLANG_VERSION=1.10.3
-ARG BUILDER_IMG=gcr.io/kubeflow-images-public/bootstrapper-builder
-ARG BUILDER_IMG_VERSION=latest
-FROM ${BUILDER_IMG}:${BUILDER_IMG_VERSION} AS build_base_remote
-# Copy the src
-COPY ./ $GOPATH/src/github.com/kubeflow/kubeflow/bootstrap/
-
-RUN go build ${GOLANG_GCFLAGS} -i -o /opt/kubeflow/bootstrapper \
-    ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/main.go
-
-FROM golang:${GOLANG_VERSION} AS build
+ARG GOLANG_VERSION=1.11.2
```

Comment:
@kunmingg Can we just delete this? Are we still using the boostrapper docker image for anything?
I think the web app has separate docker images?

Reply:
To answer my own question. The bootstrap container is what we are running for deploy.kubeflow.cloud